### ISSUE TITLE
Validation of entire workflow

### ISSUE THREAD

**Issue by coreytcallaghan** [coreytcallaghan/global_bird_abundance#5]
This one will be a bit trickier, but perhaps leave some species out of detectability model and then run imputation and estimate predicted abundance for those species. But the spatial regions no longer match (i.e., we have larger grid cells that differ to the spatial aggregations in the detectability model). So need to think about this a bit more...

**Comment by itchyshin** [comment]
@coreytcallaghan - anything I can help you with or needs somebody to talk things over - do let me know - happy to chat

**Comment by coreytcallaghan** [comment]
Thanks Shinichi. I have only been slow because of issues with rstan on the
server I use here. Then it crashed yesterday for some other reason.

I have a few questions but I want to make a bit more progress before we
catch up, so give me till the end of the week and maybe early next week we
can catch up and chat through a few things.

**Comment by itchyshin** [comment]
@coreytcallaghan no worries - OK I was suspecting running validations with brms (rstan) would be a painful and slow process (surprisingly MCMCglmm is around 20 times quicker although we cannot run the model we want to run with MCMCglmm....). OK, catch up when you are ready

**Comment by coreytcallaghan** [comment]
Yup, painful and slow, PLUS stan in the background is being annoying. We
will get there though hopefully.

If I get really stuck we can troubleshoot together.

Thanks for all the help!
